patches/tModLoader/Terraria/ModLoader/Core/ModCompile.cs

```diff
@@ -214,18 +73,19 @@ internal static void UpdateReferencesFolder()
 			if (!Directory.Exists(modReferencesPath))
 				Directory.CreateDirectory(modReferencesPath);
 
-			string tMLPath = Assembly.GetExecutingAssembly().Location;
-			string touchStamp = $"{tMLPath} @ {File.GetLastWriteTime(tMLPath)}";
+			var tMLDir = Assembly.GetExecutingAssembly().Location;
+
+			// Version checking
+			string touchStamp = BuildInfo.BuildIdentifier;
 			string touchFile = Path.Combine(modReferencesPath, "touch");
 			string lastTouch = File.Exists(touchFile) ? File.ReadAllText(touchFile) : null;
-
-			if (touchStamp == lastTouch) {
+			if (touchStamp == lastTouch && false) { // TODO remove && false. this is temp
```

Comment:
Remove `&& false` not sure why this was added, verify reference updating is working

Reply:
Resolved.